mutt/list.c

```diff
@@ -267,3 +268,29 @@ size_t mutt_list_str_split(struct ListHead *head, const char *src, char sep)
 
   return count;
 }
+
+void mutt_list_copy_tail(struct ListHead *dst, const struct ListHead *src)
```

Comment:
Please can you add a Doxygen header block for these two functions.
Also, please could you add some basic tests to `test/list/`
You'll need to alter `test/Makefile.autosetup`, `test/main.c` and copy'n'paste a couple of files in `test/list/`

Reply:
I'll try.  :-)

Reply:
Done.